### ISSUE TITLE
Androidx Fix RN 0.60, Jetifier Compliance

### ISSUE THREAD

**Issue by Jaraujo6** [Agontuk/react-native-geolocation-service#91]
In order to support React Native 0.60, library maintainers need to upgrade their android files to support androidx. This change can make a library unusable for users who have not upgraded their own app for androidx. The consensus solution for this issue is to update the build.gradle so that a library like jetifier can replace the non-compliant code.

For this app to be jetifier compliant, the following code in build.gradle needs to change from

```
compileOnly('com.facebook.react:react-native:+') 
```
to
```
implementation('com.facebook.react:react-native:+') {
        exclude group: 'com.android.support'
    }
``` 
Afterwards, all users need to do is run npx jetify to update the code before running ```react-native run-android```
The link below is an example of a successful PR implementing the above. 
https://github.com/react-native-community/react-native-maps/commit/62bb953c19e55adb29200ecff9981eb5e594a7b3?diff=unified

@Agontuk could you please add these changes to your next commit? I am unable to submit a PR to address this issue myself.

**Comment by manu006** [comment]
Same issue arises to me when I upgrade my react-native version.
This issue is arising due to react-native update for support of AndroidX

Go to 
`~/node_modules/react-native-geolocation-service/android/build.gradle`

and do the changes on 
`dependencies {
    ...
    compileOnly "com.facebook.react:react-native:+"
}`

to 

`dependencies {
    ...
    implementation('com.facebook.react:react-native:+')
}
`

**Comment by Agontuk** [comment]
I've pushed a fix for RN60 in a separate branch [https://github.com/Agontuk/react-native-geolocation-service/tree/rn60-fix](https://github.com/Agontuk/react-native-geolocation-service/tree/rn60-fix). Can you try and see if it works ?

**Comment by AilsonFreire** [comment]
I am trying to install this library and it simply crashes when build, and do not give me any error messages. I tried to install the above branch.

**Comment by IShotTheSheriff** [comment]
Thank you @Agontuk I can confirm that in my project (RN 0.60.0) this fix did make a difference (I get compile error in `2.0.1` and it compiles fine when on `rn60-fix` branch).

**Comment by oakis** [comment]
> Thank you @Agontuk I can confirm that in my project (RN 0.60.0) this fix did make a difference (I get compile error in `2.0.1` and it compiles fine when on `rn60-fix` branch).

You can now use version 3.0.0 from npm.

**Comment by Agontuk** [comment]
Closing this since the issue is fixed in 3.0.0.
